<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QDialog" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="weather">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="festival">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="5">
         <widget class="QLabel" name="pic">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="wind">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QLabel" name="week">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="city"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="temp">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="month">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="font">
          <property name="text">
           <string>更换字体</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="nongli">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="query">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="color">
          <property name="text">
           <string>更换背景颜色</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="music">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>img/播放.png</normaloff>img/播放.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QCalendarWidget::SingleSelection</enum>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::ShortDayNames</enum>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::ISOWeekNumbers</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="tip">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>备忘录</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="remember"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
